Gopkg.toml

```diff
@@ -4,8 +4,8 @@
   unused-packages = true
 
 [[constraint]]
+  branch = "feat/payments"
```

Comment:
lets merge https://github.com/RTradeLtd/config/pull/1 first, put in a release, and change this constraint back to a version constraint

Reply:
Sounds good :ok_hand: